bitsail-connectors/connector-ftp/src/main/java/com/bytedance/bitsail/connector/ftp/core/config/FtpConfig.java

```diff
@@ -0,0 +1,93 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.bytedance.bitsail.connector.ftp.core.config;
+
+import com.bytedance.bitsail.common.configuration.BitSailConfiguration;
+import com.bytedance.bitsail.connector.ftp.error.FtpErrorCode;
+import com.bytedance.bitsail.connector.ftp.option.FtpReaderOptions;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.io.Serializable;
+
+@Setter
+@Getter
+public class FtpConfig implements Serializable {
+
+  private String username;
+
+  private String password;
+
+  private String[] paths;
+
+  private String host;
+
+  private Integer port;
+
+  private Boolean skipFirstLine;
+
+  private int timeout;
+
+  private Protocol protocol;
+
+  private ConnectPattern connectPattern;
+  private ContentType contentType;
+
+  private int maxRetryTime;
+
+  private int retryIntervalMs;
```

Comment:
`successFileRetryIntervalMs` sounds better to me

Reply:
I will update it in next revision.